pkg/mapfs/fs.go

```diff
@@ -188,6 +188,7 @@ func (m *FS) RemoveAll(path string) error {
 
 func cleanPath(path string) string {
 	path = filepath.Clean(path)
+	path = strings.TrimPrefix(path, filepath.VolumeName(path)) // Remove the volume name
```

Comment:
for my understanding: I'm just curious, why do we have to do this?

Reply:
On Windows, `filepath.Clean` changes `C:` to `C:.`.

>If the result of this process is an empty string, Clean returns the string ".".

https://github.com/golang/go/blob/master/src/path/filepath/path.go#L96-L104

Then, it will not match the file path in mapfs. Since mapfs has file paths as map keys, it will be an exact match. `C:` and `C:.` will not match.

Reply:
But after thinking for a while, I changed my mind to keep the volume name as well. I've updated the logic. Thanks for giving me a chance to rethink.